### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 17

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#343] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325320-ccfb3205-ae5c-41fb-b8d6-5a5604321855.png)

**Comment by i0nut02** [comment]
<img width="491" alt="Schermata 2022-02-05 alle 11 44 00" src="https://user-images.githubusercontent.com/99051485/152638909-815c4d67-042f-4203-b418-216c9a6704ac.png">

**Comment by Exyss** [comment]
![Boh(20)](https://user-images.githubusercontent.com/65043207/152685816-36d147ca-5706-410d-a1ba-6d077f569417.jpg)

**Comment by LuginiAndrea** [comment]
$$z \implies d$$
$$\neg d$$
$$\neg z$$
Per contrapposizione, $z \implies d \iff \neg d \implies \neg z$. Sapendo che $\neg d = 1$, ne consegue che $\neg z = 1$.
Ne segue che la terza è conseguenza logica delle due proposizioni iniziali

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217812826-c2d269de-b219-49af-a822-2d851f2043b0.png)
